presto-docs/src/main/sphinx/release/release-0.245.rst

```diff
@@ -4,6 +4,7 @@ Release 0.245
 
 .. warning::
     There is a bug causing failure at startup if function namespace manager is enabled and Thrift is not configured (:pr:`15501`).
+    There is a bug causing failure when reading ORC files with ``ARRAY``/``MAP``/``ROW`` of ``VARCHAR`` columns using the selective stream readers for some corner cases (:pr:`15549`)
```

Comment:
I think we normally add separate warnings for different items (see 239 and 243 release notes). So add another `.. warning::` to this one.